frontend/src/SelectMode/SelectMode.jsx

```diff
@@ -2,17 +2,42 @@ import React from 'react';
 //import PropTypes from 'prop-types';
 //import classNames from './selectMode.module.css';
 import {Link} from "react-router-dom";
-import { Button } from 'antd';
-
+//import { Button } from 'antd';
+import "../../node_modules/antd/dist/antd.css";
```

Comment:
I *think* you can replace this import with `import "antd/dist/antd.css";`, but test before to be sure